### PR TITLE
337 Ims node select and location code fields

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/ImsNodeIdField.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                    }
+                })
+                .catch((error) => {
+                    console.error(error);
+                })
+                .finally(() => setIsLoading(false));
+        }
+    }, [locationCode, status, axiosApiClient, unsubscribedOnly]);
+
+    const placeholder =
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoForms/formFields/ImsNodeIdField.tsx:80]: The : ? : ? is hard to follow, assume its a copy paste from v1?

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/ImsNodeIdField.tsx (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoForms/formFields/ImsNodeIdField.tsx:80]: Yes it is. I tried to format it a little better but prettier has a strong opinion this is how it should be

File: packages/orchestrator-ui-components/src/components/WfoForms/formFields/OrganisationField.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ export type OrganisationFieldProps = Omit<
 >;
 
 function Organisation({ ...props }: OrganisationFieldProps) {
-    const t = useTranslations('forms');
+    const t = useTranslations('pydanticForms');
```
Comment (acidjunk) [inline packages/orchestrator-ui-components/src/components/WfoForms/formFields/OrganisationField.tsx:28]: Hmm, this reminds me about translations. Which is a backend thing for v1. E.g. form labels and description texts.
Not sure if we implemented this already.

**Review by acidjunk — APPROVED** [review]
